app/views/admin/configs/_tab_others.html.haml

```diff
@@ -1,5 +1,9 @@
 = config_input form, :use_nick, as: :boolean
 = config_input form, :tolerance_is_costly, as: :boolean
+- distribution_options = [[t('config.keys.distribution_strategy_options.first_order_first_serve'), FoodsoftConfig::DistributionStrategy::FIRST_ORDER_FIRST_SERVE],
+  [t('config.keys.distribution_strategy_options.no_automatic_distribution'), FoodsoftConfig::DistributionStrategy::NO_AUTOMATIC_DISTRIBUTION]]
+= config_input form, :distribution_strategy, as: :select, collection: distribution_options,
```

Comment:
Can we use `label_method` here? Doing it like [here](https://github.com/foodcoops/foodsoft/blob/4043433539f34b880bec8826d2f29f9137412ef4/app/views/finance/balancing/_edit_transport.html.haml#L7-L8) would make the code a little bit more readable.

Reply:
Sure - good point :ok_hand: